site/content/en/docs/Installation/Creating Cluster/aks.md

```diff
@@ -27,8 +27,15 @@ AKS_LOCATION=westeurope          # Azure region in which you'll deploy your AKS
 az group create --name $AKS_RESOURCE_GROUP --location $AKS_LOCATION
 
 # Create the AKS cluster - this might take some time. Type 'az aks create -h' to see all available options
+
+{{% feature expiryVersion="1.9.0" %}}
```

Comment:
Can you double check how this renders? Last time I tried this I don't think it worked to insert these inside of the triple backticks.

Reply:
It renders a giant blank space inside of the gray box: https://9723542-dot-preview-dot-agones-images.appspot.com/site/docs/installation/creating-cluster/aks/

Reply:
Yes, let me fix the render.